chess-AI/train.py

```diff
@@ -74,7 +73,7 @@ def training_game(self):
             # If the game is over, end the episode
             # TODO: Consider removing `board.can_claim_draw()` as it may be slow to check.
             # See https://python-chess.readthedocs.io/en/latest/core.html#chess.Board.can_claim_draw
-            if board.is_game_over() or board.can_claim_draw():
+            if board.is_game_over() or board.can_claim_fifty_moves() or board.can_claim_threefold_repetition() or board.is_seventyfive_moves():
```

Comment:
is this for sure necessary? everything else looks good

Reply:
can_claim_draw() performs the same function as can_claim_fifty_moves and can_claim_threefold_repetition(), so we could technically keep that, but it doesn't include the 75 move rule. If we want the game to stop as a draw from the 75 move rule we would need this part.

Reply:
isnt the 75 move rule a subset of the 50 move rule?

Reply:
actually yeah 75 move rule is pointless if we check for 50 move rule, I'll get rid of it